ppr-ui/src/components/mhrTransfers/ConfirmCompletion.vue

```diff
@@ -28,7 +28,66 @@
-              <ol>
+              <ol
+                v-if="isTransportPermit"
+                id="confirm-transport-permit"
+              >
+                <li>
+                  <strong>Current home location and homeowner(s) named in the Manufactured Home Registry match</strong>
+                  the current home location and ownership of the home on the Transport Application form.
+                  <v-divider class="my-6 ml-n8" />
+                </li>
+                <li>
+                  <strong>A
+                    <v-tooltip
+                      location="top"
+                      contentClass="top-tooltip pa-5"
+                      transition="fade-transition"
+                    >
+                      <template #activator="{ props }">
+                        <span
+                          class="underline"
+                          v-bind="props"
+                        >tax collector’s clearance certificate for transport purposes</span>
+                      </template>
+                      A certificate or confirmation in electronic format that confirms that no
+                      property taxes are unpaid. Reference Manufactured Home Act sections 25 and 26(2).
+                    </v-tooltip></strong>
+                  showing that all local taxes for the current year have been paid is required for
+                  manufactured homes that:
+                  <ol
+                    type="a"
+                  >
+                    <li>
+                      are not located on a manufacturer’s or a dealer's lot.
+                    </li>
+                    <li>are not moving to a different pad within the same park.</li>
+                  </ol>
+                  <v-divider class="my-6 ml-n8" />
+                </li>
+                <li>
+                  <strong>Personal Property Registry search has been completed</strong> and there are no liens that
+                  block the transport permit. PPR registrations that block a transport permit include the following:
+                  <ul>
+                    <li>Land Tax Deferment Lien on a Manufactured Home</li>
+                    <li>Maintenance Lien</li>
+                    <li>Manufactured Home Notice</li>
+                  </ul>
+                  <v-divider class="my-6 ml-n8" />
+                </li>
+                <li>
+                  <strong>A report must be submitted to the Registrar</strong> within 3 days of expiry if either of
+                  the following apply:<br><br>
+                  <ol type="a">
+                    <li>The manufactured home is not moved before the transport permit expires</li>
+                    <li>
+                      The home is permanently moved to a different location than what is specified on
+                      the transport permit
+                    </li>
+                  </ol>
+                </li>
+              </ol>
+              <ol v-else>
```

Comment:
Wouldn't this conditional content be applied using the slot from the parent vs in the slot as a default with a condition?

Reply:
Yea I was looking into it as well. We would have to put a condition in the parent component, because the parent `ConfirmCompletion` has only one instance for Transfers and Permits. So I decided to have a condition within, to keep `MhrInformation` a bit cleaner (from this chunk of static content).

Reply:
As above, i know its a time crunch, you could flag this as something to refactor in a follow up if you prefer. 
Does seem like this content isn't default content and should be applied through the slot but you can update later if you prefer.

Reply:
Ideally I wanted to have this html content in a resource file, but that requires additional configurations, so I dropped it for now. I'll leave as is and mark for future refactor. Thanks!